include/FileBrowser.h

```diff
@@ -137,14 +140,18 @@ class FileBrowserTreeWidget : public QTreeWidget
 
 	bool m_mousePressed;
 	QPoint m_pressPos;
-
+	
 	//! This should only be accessed or modified when m_pphMutex is held
 	PlayHandle* m_previewPlayHandle;
+	
+#if (QT_VERSION >= QT_VERSION_CHECK(5,14,0))
+	QRecursiveMutex m_pphMutex;
+#else
 	QMutex m_pphMutex;
+#endif
 
 	QList<QAction*> getContextActions(FileItem* item, bool songEditor);
 
-
```

Comment:
Is this wanted?

Reply:
Although I don't entirely agree with the formatting here, I'll fix it so it can be consistent with the spacing from the other access specifiers.